server/datastore/mysql/apple_mdm.go

```diff
@@ -640,7 +641,12 @@ func upsertMDMAppleHostDisplayNamesDB(ctx context.Context, tx sqlx.ExtContext, h
 	return nil
 }
 
-func upsertMDMAppleHostMDMInfoDB(ctx context.Context, tx sqlx.ExtContext, serverURL string, fromSync bool, hostIDs ...uint) error {
+func upsertMDMAppleHostMDMInfoDB(ctx context.Context, tx sqlx.ExtContext, serverSettings fleet.ServerSettings, fromSync bool, hostIDs ...uint) error {
```

Comment:
I made the argument of type `fleet.ServerSettings` so it is not ambiguous/unclear whether the string should be just the Fleet server URL or the fully resolved Fleet MDM URL.